torch/_subclasses/fake_tensor.py

```diff
@@ -1316,6 +1423,13 @@ def tolist(self):
 
 
 class FakeTensorMode(TorchDispatchMode):
+    cache: Dict[_DispatchCacheKey, _DispatchCacheEntry] = {}
+    cache_hits: int = 0
+    cache_misses: int = 0
+    cache_bypasses = defaultdict(int)
+    cache_enabled: bool = FakeTensorConfig.cache
+    cache_crosscheck: bool = FakeTensorConfig.cache_crosscheck
```

Comment:
What's the point of having `FakeTensorConfig` versions of these variables if they're only read at import time?

Reply:
oops - Left over from an earlier impl where the settings weren't class variables in FakeTensorConfig. I'll fix!